### DISCUSSION TITLE
AR(1) Spatio-temporal Random Fields

### DISCUSSION THREAD

**Discussion by Nathan-Hebert** [sdmTMB/sdmTMB#183] [Q&A] [answered]
![Capture](https://user-images.githubusercontent.com/62037477/222201038-1fedfce8-8716-40c9-90ba-82c3ba5bcd44.png)

I'm not sure I understand the purpose of the sqrt(1-rho^2) bit. What is the steady-state marginal variance? Thanks!

**Comment by ericward-noaa** [comment]
I'm not sure I understand the question -- but here's a little more info on the TMB implementation:

https://kaskr.github.io/adcomp/classdensity_1_1AR1__t.html

And even in a simpler scenario, like a 1-D time series setting, we'd also want to do the same scaling

**Comment by Nathan-Hebert** [comment, marked as answer]
I guess what I'm asking is, if we had only epsilon_t as the second term rather than sqrt(1-rho^2)*epsilon_t, what would happen to our fields?

**Reply by ericward-noaa** [reply]
I"ll try to illustrate this with the time series version because it's easier to see. The short version is without the adjustment, the variance blows up because it is a function of $\rho$. 

In the first case, let's say we have a correlated random walk model, 

$x[0] \sim N(0, \sigma)$ 
$x[1] = \rho * x[0] + N(0, \sigma)$
$x[2] = \rho * x[1] + N(0, \sigma)$

So Var(x[0]) is $\sigma^2$, but Var(x[1]) = $\rho^2 * \sigma^2 + \sigma^2$, and it keeps ballooning with x[2], x[3], ...

In the second case, we can do the same thing with the adjustment built in

$x[0] \sim N(0, \sigma)$ 
$x[1] = \rho * x[0] + N(0, sqrt(1 - \rho^2) * \sigma)$ 

As before, Var(x[0]) = $\sigma^2$, but now Var(x[1]) = $\rho^2 * \sigma^2 + (1 - \rho^2) * \sigma^2 = \sigma^2$, and
all other elements of x will have the same variance = $\sigma^2$.

**Comment by Nathan-Hebert** [comment]
Thank you so much! That makes perfect sense now. :)
